msfvenom

```diff
@@ -324,19 +365,30 @@ if generator_opts[:list]
     case mod.downcase
     when "payloads", "payload", "p"
       $stdout.puts dump_payloads
+#    when "options", "option", "o"
+#      opts[:list_options] = true
```

Comment:
This looks to remove the generic options global list, was that intended?  If yes then remove the commented lines, also please provide a reasoning for why this should be removed.

Reply:
It was a feature that I wanted to add, but never finished - nor removed when committing !

Reply:
Removed.
Added to wish list